diagnostic.md

```diff
@@ -92,6 +109,9 @@ div {
 ```
 
 <!-- your answer starts here -->
+The div labeled 'specialDiv' will have the following values:
+Text color ==>  yellow
+font-family ==> sans-serif
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive.